orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RollingRedBlackStrategy.groovy

```diff
@@ -65,24 +65,11 @@ class RollingRedBlackStrategy implements Strategy, ApplicationContextAware {
         cloudProvider                          : cleanupConfig.cloudProvider,
     ]
 
-    Map originalCapacity = stageData.originalCapacity ?: stageData.capacity
```

Comment:
Realized that this could be problematic if someone ever RRB'd w/o `use source capacity` ... this seems like an abnormal use case. 

Is there a strong desire to maintain this?

RRB'ing to a fixed capacity doesn't seem to jive with disabling certain percentages of an existing server group / cluster?

Reply:
Oh I think this is here because for a reason I could never find - the strategy is evaluated twice with the same stage data. I could be wrong about this being here for that reason but I would double check.

Reply:
I'm not sure we ever need to care about capacity stored on the stage data ... I reckon the UI would let you set an explicit capacity right now but that doesn't make a lot of sense to me? 

Would be problematic if any autoscaling had occurred (which is the motivation behind making all this runtime / dynamic).

Reply:
I think the capacity is coming from whatever the capacity is when the stage starts - but you're right - it should be updated to reflect any scaling

Reply:
This PR reflects that.

Reply:
heh yup was agreeing with you